packages/terra-framework-docs/src/terra-dev-site/doc/data-grid/FlowsheetDataGrid.4/Examples.6/ColumnHeadersHidden.4.doc.mdx

```diff
@@ -0,0 +1,21 @@
+import ColumnHeadersHidden from './ColumnHeadersHidden?dev-site-example';
+
+# Hiding Column Headers
+
+### Description
+The [Flowsheet Data Grid](/components/cerner-terra-framework-docs/data-grid/flowsheet-data-grid/about) can hide column headers with the `hasVisibleColumnHeaders` prop.
+
+### Usage
+The `hasVisibleColumns` is a boolean. By default, it is set to `true`.
+When it is set to `false`, the column headers are hidden, but still exist in the DOM and can be read by screenreaders when providing context for the table.
+This is useful when needing to use custom alternative to column headers, such as Stella Timeline.
+
+**Note:**
+- Column header labels should still be provided in the dataset to ensure that sceenreaders will still read out the appropriate context for the table.
+- When using custom column headers, then logic must be added to ensure that the column widths match are in sync if resizing columns are enabled.
+The [columns.width](/components/cerner-terra-framework-docs/data-grid/flowsheet-data-grid/about#column) property for columns can be used to set the column widths programatically.
```

Comment:
It seems like this should be on the About page instead of an example page.  You would not want consumers to miss this information because they did not select one of the examples.

Reply:
For now, I'm just adding the necessary information. It can be reorganized in the implementation guide story which will be updating the documentation as a whole and making sure all pieces fit together.